examples/app-with-extensions/provisioning/plugins/app.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: 1
+
+apps:
+  - type: myorg-extensions-app
+    org_id: 1
+    org_name: Main Org.
+    disabled: false
```

Comment:
Is this the provisioning equivalent of `enabled: true`? 🤔

Reply:
Yes - so weird